### ISSUE TITLE
selection option {{tl|Check categories}}

### ISSUE THREAD

**Issue by OpenGreenStreet** [commons-app/apps-android-commons#5518]
### What is the user problem or growth opportunity you want to see solved?

Check categories: Finding the right category in the Commons app is much more time-consuming - both in terms of content and time - than on the desktop. Finding the right category for a (main) location, for example, is easy, but then finding the right subcategory quickly becomes complicated using the Commons app . . .

An alternative would be to set {{tl|Check categories}}. For this, there should be a selection option for {{tl|Check categories}} when uploading.

### How do you know that this problem exists today? Why is this important?

Improved convenience and higher quality of categorization

### Who will benefit from it?

wikimedia commons

### Anything else you would like to add?

https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback#Feedback_from_Molgreen_for_version_4.2.1~14b6c455b

**Comment by nicolas-raoul** [comment]
Good idea, in the spirit of OpenStreetMap's "please someone check my edits".

**Comment by OpenGreenStreet** [comment]
@nicolas-raoul : I like the Commons app and there is a second app that I like "StreetComplete". Maybe the two apps could learn from each other.
* https://wiki.openstreetmap.org/wiki/DE:StreetComplete
* https://streetcomplete.app/
